### PR TITLE
[HOSTING-383] - Create plugin-azure-cli for @torosent

### PR DESCRIPTION
**Pull request by torosent**
# Description

Hosting: https://issues.jenkins-ci.org/projects/HOSTING/issues/HOSTING-383?filter=allissues
GitHub: https://github.com/jenkinsci/azure-cli-plugin

# Submitter checklist for changing permissions

### Always

- [x] Add link to plugin/component Git repository in description above

### For a newly hosted plugin only

- [x] Add link to resolved HOSTING issue in description above

### For a new permissions file only

- [x] Make sure the file is created in `permissions/` directory
- [x] `artifactId` (pom.xml) is used for `name` (permissions YAML file).
- [x] [`groupId` / `artifactId` (pom.xml) are correctly represented in `path` (permissions YAML file)](https://github.com/jenkins-infra/repository-permissions-updater/#managing-permissions)
- [x] Check that the file is named `plugin-${artifactId}.yml` for plugins

### CODE REVIEW

**Comment by daniel-beck** [comment]
@torosent Please re-read the checklist items and actually follow their instructions. Thanks!

**Comment by torosent** [comment]
changed the list and the description. thanks

File: permissions/plugin-azure-cli.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+name: "azure-cli"
+paths:
+- "org/jenkins-ci/plugins/azure-cli"
```
Comment (daniel-beck) [inline permissions/plugin-azure-cli.yml]: This seems wrong: https://github.com/jenkinsci/azure-cli-plugin/blob/master/pom.xml#L11

Please see the checklist for a reference to the docs in this repo on how this works.

**Comment by torosent** [comment]
updated pom.xml

File: permissions/plugin-azure-cli.yml (hunk 1 above)
Comment (torosent) [inline permissions/plugin-azure-cli.yml]: updated pom.xml

**Pull request merged**